package.json

```diff
@@ -29,7 +29,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-vue": "^8.0.3",
-    "jest": "^28.1.2",
+    "jest": "^27.5.1",
```

Comment:
Why the downgrade?

Reply:
Without I could not import modules, because other dependencies that are needed are also only version 27.x and I could not upgrade these.